.circleci/config.yml

```diff
@@ -187,6 +189,11 @@ references:
       - run:
           name: Set cluster
           command: doctl kubernetes cluster kubeconfig save "$DOKS_CLUSTER_NAME" --set-current-context
+      - run:
+          name: "Create namespace"
+          command: |
+            kubectl delete namespace $NAMESPACE
```

Comment:
Wont you delete everything if you do this?? Including all the data (pvcs) in the k8s dev installtion?

Reply:
They only get deleted if you --purge

Reply:
purge is only a thing from helm https://v2.helm.sh/docs/helm/#helm-delete:
<img width="1100" alt="image" src="https://user-images.githubusercontent.com/3610528/70917710-ba497600-2015-11ea-9a9b-5959066d506d.png">

Reply:
kubectl dosen't have purge

Reply:
If you delete the namespace everything inside the namespace is deleted (pvcs included). I am not sure if you will get the same PVs once you create e everything again.